cvmfs/util.cc

```diff
@@ -927,15 +906,6 @@ bool HasSuffix(
 }
 
 
-bool IsNumeric(const std::string &str) {
```

Comment:
I think we have this already in the sanitizer classes. Wouldn't it make more sense to take it from there?